### PR TITLE
vnu 20170206

### PR DESCRIPTION
**Pull request by ymyzk**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/vnu.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Vnu < Formula
   desc "Nu Markup Checker: command-line and server HTML validator"
   homepage "https://validator.github.io/validator/"
-  url "https://github.com/validator/validator/releases/download/17.2.0/vnu.jar_17.2.0.zip"
-  version "20170130"
-  sha256 "9e96616caf5ec720ab6db12ea0d9a9a3fd97128551e3d358e1bb7a50d4310aa6"
+  url "https://github.com/validator/validator/releases/download/17.2.1/vnu.jar_17.2.1.zip"
+  version "20170206"
+  sha256 "e859b26b9532b4d1a5cb5c3bc873c85dc7a64c05ef7cb3f98ef040690d95609b"
```
Comment (fxcoudert) [inline Formula/vnu.rb]: The webpage says it requires Java 8, so this should be in the formula

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/vnu.rb (hunk 1 above)
Comment (fxcoudert) [inline Formula/vnu.rb]: Also: why do we number versions by date, when they have perfectly fine version numbers (17.2.1)?

**Comment by nijikon** [comment]
@fxcoudert lets see what will happen, this might be cause there was some bogus version in the past.

**Comment by fxcoudert** [comment]
@nijikon we can force the version to be explicitly `17.2.1` if we need, I was just wondering why using a date instead of the version number.

It might need a `version_scheme 1`?

**Comment by ilovezfs** [comment]
https://github.com/validator/validator/releases

The headline for the release version used to be the date. We can either continue that tradition, and make it "20170206" or switch to 17.2.1 and bump the version_scheme. The latter is probably better since it can be derived from the url automatically.

**Comment by fxcoudert** [comment]
```== /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/vnu.rb ==
C:  8:  1: Trailing whitespace detected.

vnu:
  * stable version should not decrease (from 20170130 to 17.2.1)
```

**Comment by fxcoudert** [comment]
Thanks @ymyzk for your contribution!

**Comment by ymyzk** [comment]
Thanks for fixing!

**Pull request closed without merging**